openc3-cosmos-init/plugins/packages/openc3-vue-common/src/tools/base/Login.vue

```diff
@@ -147,28 +152,33 @@ export default {
       const redirect = new URLSearchParams(window.location.search).get(
         'redirect',
       )
-      if (redirect.startsWith('/tools/')) {
+      if (redirect?.startsWith('/tools/')) {
         // Valid relative redirect URL
         window.location = decodeURI(redirect)
       } else {
         window.location = '/'
       }
     },
-    verifyPassword: function () {
+    verifyPassword: function (token, noAlert) {
```

Comment:
Is this ever called with `noAlert` set to false?

Reply:
It's called with `undefined` as discussed above